DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/CommunicationModels/Graph.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.Statistics.WebApi.Models;
+
+namespace GitHubExtension.Statistics.WebApi.CommunicationModels
+{
+    public class Graph
```

Comment:
If this Model please use post fix Model, if an entity user Entity...Because now under namespaces as I understand you are mixing entities and models.